docs/user-tutorial/benchmarks/micro-benchmarks.md

```diff
@@ -101,16 +101,17 @@ or [AMD](https://github.com/ROCm-Developer-Tools/HIP/tree/master/samples/1_Utils
 | mem-bw/D2H_Mem_BW | bandwidth (GB/s) | Device to host copy bandwidth.   |
 | mem-bw/D2D_Mem_BW | bandwidth (GB/s) | Device to device copy bandwidth. |
 
-### `gpu-sm-copy-bw`
+### `gpu-copy-bw`
```

Comment:
why change the name? did you change  it in benchmark code?

Reply:
Yes